io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications"
```

Comment:
Is this used? Electron doesn't use this for notifications

Reply:
Catalyst has no notification support yet.

Reply:
Then what is it for?

Reply:
I'm unsure. I'm not the one creating this flatpak package. I'm just accepting changes upstream and checking things

Reply:
"dri" might(?) be needed for controllers. Removed anyways